### ISSUE TITLE
Skills/Spells not notching Spellcasts/skills, Possible reason and fix included.

### ISSUE THREAD

**Issue by Zipalodok** [torilmud/issues#867]
Feel pretty confident any related skill/spell that does not notch, is missing the "Related Skill" portion.

Some spells do not notch their spellcast at all, whether cast or power version for past 3-5years now.

What I noticed is this. Take "silence person" as example, which does not notch spellcast generic. If you "power info silence per" you'll notice no "Related Skill" to it, comepletely missing. Where as spells with a "Related Skill", actually do notch and increase stated skill.

Like earthquake didn't notch spellcast nature for a decade, then 3-5 years ago it was magically fixed. When you converted it to a power. Well if you "powers info earthquake" you'll see it associated with spellcast nature. Which is my assumption to why it works.

So if the game is not even checking my spellcast generic, would explain why silence utterly sucks to land.

Feel pretty confident any related skill/spell that does not notch, is missing the "Related Skill" portion.

Hope that helps you dail down this issue. I'm sure you're aware of it, in some regard. But this seems like the direct issue with it, least with powers.

**Comment by shevarash** [comment]
You're correct that a power can't notch a spellcast skill unless it is in the `Related Skill` section. That said, silence person's effectiveness has nothing to do with your underlying spellcast skill. As you can see on the powers info, it's just s simple saving throw to resist.

That said, we should make sure the power spells are associated with the correct spellcast, so we'll use this issue for that. If anyone has seen other power spells that don't have a related skill, please feel free to post them in this issue.
